pip/basecommand.py

```diff
@@ -105,6 +105,12 @@ def parse_args(self, args):
     def main(self, args):
         options, args = self.parse_args(args)
 
+        if options.quiet and options.verbose:
```

Comment:
I think rather than doing this, it'd be better to do something like:

```python
verbosity = options.verbose - options.quiet
if verbosity >= 1:
    level = "DEBUG"
elif verbosity == -1:
    level = "WARNING"
elif verbosity == -2:
    level = "ERROR"
elif verbosity <= -3:
    level = "CRITICAL"
else:
    level = "INFO"
```

This eliminates the warning (which kind of defeat the purpose of the ``-q`` flag) and allows combinations of ``-q`` and ``-v`` to act sanely (each use of ``-q`` lowers the verbosity level until it gets max quiet, each use of ``-v`` raises it until it gets max noisy, combinations of ``-q`` and ``-v`` end up being combined to determine the output level).